### ISSUE TITLE
Trying to view vehicle status gives 'Internal server error'

### ISSUE THREAD

**Issue by korhojoa** [tillsteinbach/VWsFriend#366]
Hi, I have two e-Golfs, 2014 and 2019.
2014 seems to work, 2019 reports this when trying to open the status page.
Perhaps the climatization status isn't present?
https://github.com/tillsteinbach/VWsFriend/blob/main/vwsfriend/vwsfriend/ui/templates/status/vehicle.html#L243
```
vwsfriend-vwsfriend-1   | 2022-11-19T10:32:16+0000:INFO:_internal:192.168.1.243 - - [19/Nov/2022 10:32:16] "GET /status/vehicle/WVWZZZAUZKW000000/ HTTP/1.1" 500 -
vwsfriend-vwsfriend-1   | 2022-11-19T10:32:16+0000:ERROR:_internal:Error on request:
vwsfriend-vwsfriend-1   | Traceback (most recent call last):
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/werkzeug/serving.py", line 335, in run_wsgi
vwsfriend-vwsfriend-1   |     execute(self.server.app)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/werkzeug/serving.py", line 322, in execute
vwsfriend-vwsfriend-1   |     application_iter = app(environ, start_response)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask/app.py", line 2548, in __call__
vwsfriend-vwsfriend-1   |     return self.wsgi_app(environ, start_response)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask/app.py", line 2528, in wsgi_app
vwsfriend-vwsfriend-1   |     response = self.handle_exception(e)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
vwsfriend-vwsfriend-1   |     response = self.full_dispatch_request()
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
vwsfriend-vwsfriend-1   |     rv = self.handle_user_exception(e)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
vwsfriend-vwsfriend-1   |     rv = self.dispatch_request()
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
vwsfriend-vwsfriend-1   |     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask_login/utils.py", line 290, in decorated_view
vwsfriend-vwsfriend-1   |     return current_app.ensure_sync(func)(*args, **kwargs)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/vwsfriend/ui/status.py", line 25, in vehicle
vwsfriend-vwsfriend-1   |     return render_template('status/vehicle.html', vehicle=vehicles[vin], current_app=current_app)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask/templating.py", line 147, in render_template
vwsfriend-vwsfriend-1   |     return _render(app, template, context)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/flask/templating.py", line 130, in _render
vwsfriend-vwsfriend-1   |     rv = template.render(context)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
vwsfriend-vwsfriend-1   |     self.environment.handle_exception()
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
vwsfriend-vwsfriend-1   |     raise rewrite_traceback_stack(source=source)
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/vwsfriend/ui/templates/status/vehicle.html", line 1, in top-level template code
vwsfriend-vwsfriend-1   |     {% extends 'base.html' %}
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/vwsfriend/ui/templates/base.html", line 40, in top-level template code
vwsfriend-vwsfriend-1   |     {% block content %}{% endblock %}
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/vwsfriend/ui/templates/status/vehicle.html", line 243, in block 'content'
vwsfriend-vwsfriend-1   |     {{vehicle.domains['climatisation']['windowHeatingStatus'].carCapturedTimestamp.value.isoformat()}})</h2>
vwsfriend-vwsfriend-1   |   File "/opt/venv/lib/python3.10/site-packages/jinja2/utils.py", line 83, in from_obj
vwsfriend-vwsfriend-1   |     if hasattr(obj, "jinja_pass_arg"):
vwsfriend-vwsfriend-1   | jinja2.exceptions.UndefinedError: 'None' has no attribute 'isoformat'
```

EDIT: Workaround is to set the window heating once, then the page can load.

**Comment by tillsteinbach** [comment]
Thanks for reporting. Will be fixed in the next release

**Comment by tillsteinbach** [comment]
Should be fixed in 0.23.0
